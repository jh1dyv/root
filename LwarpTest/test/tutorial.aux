\relax 
\providecommand\hyper@newdestlabel[2]{}
\providecommand\HyperFirstAtBeginDocument{\AtBeginDocument}
\HyperFirstAtBeginDocument{\ifx\hyper@anchor\@undefined
\global\let\oldcontentsline\contentsline
\gdef\contentsline#1#2#3#4{\oldcontentsline{#1}{#2}{#3}}
\global\let\oldnewlabel\newlabel
\gdef\newlabel#1#2{\newlabelxx{#1}#2}
\gdef\newlabelxx#1#2#3#4#5#6{\oldnewlabel{#1}{{#2}{#3}}}
\AtEndDocument{\ifx\hyper@anchor\@undefined
\let\contentsline\oldcontentsline
\let\newlabel\oldnewlabel
\fi}
\fi}
\global\let\hyper@last\relax 
\gdef\HyperFirstAtBeginDocument#1{#1}
\providecommand\HyField@AuxAddToFields[1]{}
\providecommand\HyField@AuxAddToCoFields[2]{}
\@writefile{toc}{\contentsline {chapter}{\numberline {1}はじめに}{7}{chapter.1}}
\@writefile{lof}{\addvspace {10\p@ }}
\@writefile{lot}{\addvspace {10\p@ }}
\@writefile{toc}{\contentsline {chapter}{\numberline {2}First chapter}{9}{chapter.2}}
\@writefile{lof}{\addvspace {10\p@ }}
\@writefile{lot}{\addvspace {10\p@ }}
\@writefile{toc}{\contentsline {chapter}{\numberline {3}C\#について}{11}{chapter.3}}
\@writefile{lof}{\addvspace {10\p@ }}
\@writefile{lot}{\addvspace {10\p@ }}
\@writefile{toc}{\contentsline {section}{\numberline {3.1}A section}{11}{section.3.1}}
\@writefile{toc}{\contentsline {section}{\numberline {3.2}C\#について さらに}{11}{section.3.2}}
\@writefile{toc}{\contentsline {subsection}{\numberline {3.2.1}C\#について もう一度}{11}{subsection.3.2.1}}
\@writefile{toc}{\contentsline {subsubsection}{C\#について さらに詳しく}{11}{section*.3}}
\@writefile{lof}{\contentsline {figure}{\numberline {3.1}{\ignorespaces A figure with text}}{11}{figure.3.1}}
\newlabel{fig:withtext}{{3.1}{11}{A figure with text}{figure.3.1}{}}
\newlabel{fig:withtext@cref}{{[figure][1][3]3.1}{[1][11][]11}}
\@writefile{toc}{\contentsline {section}{\numberline {3.3}Some math}{12}{section.3.3}}
