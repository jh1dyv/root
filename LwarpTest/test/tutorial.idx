\indexentry{Some text.|hyperpage}{11}
\indexentry{漢字の索引|hyperpage}{11}
