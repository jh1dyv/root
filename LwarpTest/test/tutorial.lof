\addvspace {10\p@ }
\addvspace {10\p@ }
\addvspace {10\p@ }
\contentsline {figure}{\numberline {3.1}{\ignorespaces A figure with text}}{11}{figure.3.1}
