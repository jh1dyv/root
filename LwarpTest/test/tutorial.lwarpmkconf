confversion = [[1]]
opsystem = [[Windows]]
sourcename = [[tutorial]]
homehtmlfilename = [[tutorial]]
htmlfilename = [[]]
latexmk = [[false]]
printlatexcmd = [[pdflatex tutorial]]
HTMLlatexcmd = [[pdflatex tutorial_html]]
printindexcmd = [[makeindex -s lwarp.ist tutorial.idx]]
HTMLindexcmd = [[makeindex -s lwarp.ist tutorial_html.idx]]
latexmkindexcmd = [[makeindex -s lwarp.ist]]
glossarycmd = [[makeglossaries]]
pdftotextenc = [[UTF-8]]
