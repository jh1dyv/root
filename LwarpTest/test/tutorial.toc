\contentsline {chapter}{\numberline {1}はじめに}{7}{chapter.1}
\contentsline {chapter}{\numberline {2}First chapter}{9}{chapter.2}
\contentsline {chapter}{\numberline {3}C\#について}{11}{chapter.3}
\contentsline {section}{\numberline {3.1}A section}{11}{section.3.1}
\contentsline {section}{\numberline {3.2}C\#について さらに}{11}{section.3.2}
\contentsline {subsection}{\numberline {3.2.1}C\#について もう一度}{11}{subsection.3.2.1}
\contentsline {subsubsection}{C\#について さらに詳しく}{11}{section*.3}
\contentsline {section}{\numberline {3.3}Some math}{12}{section.3.3}
