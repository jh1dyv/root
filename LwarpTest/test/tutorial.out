\BOOKMARK [0][-]{chapter.1}{\060\157\060\130\060\201\060\153}{}% 1
\BOOKMARK [0][-]{chapter.2}{First chapter}{}% 2
\BOOKMARK [0][-]{chapter.3}{C\043\060\153\060\144\060\104\060\146}{}% 3
\BOOKMARK [1][-]{section.3.1}{A section}{chapter.3}% 4
\BOOKMARK [1][-]{section.3.2}{C\043\060\153\060\144\060\104\060\146 \060\125\060\211\060\153}{chapter.3}% 5
\BOOKMARK [2][-]{subsection.3.2.1}{C\043\060\153\060\144\060\104\060\146 \060\202\060\106\116\000\136\246}{section.3.2}% 6
\BOOKMARK [1][-]{section.3.3}{Some math}{chapter.3}% 7
